crates/burn-import/onnx-tests/build.rs

```diff
@@ -54,6 +54,8 @@ fn main() {
         .input("tests/pow/pow.onnx")
         .input("tests/pow/pow_int.onnx")
         .input("tests/unsqueeze/unsqueeze.onnx")
+        .input("tests/unsqueeze/unsqueeze_opset16.onnx")
```

Comment:
See next comment (opset should be 13 not 16)

Reply:
See my above answer.